google/datalab/contrib/mlworkbench/_local_predict.py

```diff
@@ -88,9 +94,14 @@ def _download_images(data, img_cols):
   for d in data:
     for img_col in img_cols:
       if d.get(img_col, None):
-        with file_io.FileIO(d[img_col], 'rb') as fi:
-          im = Image.open(fi)
-        images[img_col].append(im)
+        if isinstance(d[img_col], Image.Image):
```

Comment:
I know. I think this path will only run if the user does

%%ml predict
prediction_data: $df_with_images

I am worried this is confusing: local prediction data looks different from cloud prediction data.